### ISSUE TITLE
[XCCDF] Expansion of "warning"

### ISSUE THREAD

**Issue by balleman-ctr** [usnistgov/SCAP#19]
Presently the xccdf:warning element can be used in Rule, Value, and Group elements.  Sometimes warnings may be targeted to a specific check or fix.  We can include these warnings at the Rule level, but it could be more accurate to include them within a check or fix element.
Profiles are another location where warnings may be of value, warning against conditions inappropriate for selecting the profile.
Additionally, the warning element supports the category attribute with a list of allowed values.  Instead of adding check or fix as allowed locations for warnings, they could be added as allowed categories.  In the case of a warning for a check, a category of “false positive” or “false negative” could be useful in further categorizing the nature of the warning.

**Comment by vanderpol** [comment]
@balleman-ctr I agree with this, but it might be useful to provide an example, or a branch to the proposed schema changes like you did with the checktext proposal.

**Comment by balleman-ctr** [comment]
I've created proposed changes and example content at https://github.com/balleman-ctr/xccdf/pull/3/files.  I did not include changes for _fix_; in my view more would need done first to structure _fix_ more like _check_ before it would make sense to include _warning_ there.

**Comment by vanderpol** [comment]
@balleman-ctr great!  Now we just need to get the source XCCDF XSD file committed to github so we can branch it and include that change in a PR.  @dragosprisaca hoping you can add all of the relevant XSD files (minus OVAL) to this repository very soon?
